src/models/auditModelGenerator.js

```diff
@@ -0,0 +1,149 @@
+// Load in our dependencies
+import { Model, DataTypes } from 'sequelize'; // eslint-disable-line import/no-import-module-exports
+import httpContext from 'express-http-context'; // eslint-disable-line import/no-import-module-exports
+import { auditLogger } from '../logger'; // eslint-disable-line import/no-import-module-exports
+
+const dmlType = ['INSERT', 'UPDATE', 'DELETE'];
+
+// const exception = () => {
```

Comment:
Can remove

Reply:
its used in the model definition, should I just define it in the model?

Reply:
This commented out code is used in the model definition?